### ISSUE TITLE
getDevice rename

### ISSUE THREAD

**Issue by harrisonhjones** [harrisonhjones/phpParticle#18]
Should be called something like addDevice or claimDevice

"get" sounds like it's supposed to get device details

**Comment by devinpearson** [comment]
I agree with this. I would think its better to rename the functions similar to spark-cli functions

**Comment by harrisonhjones** [comment]
Feel free to go on a renaming spree. If you get time I would love a way to easily produce docs from the function headers/documentation you have put in. I've never done automatic doc production. Perhaps you know of some?

**Comment by harrisonhjones** [comment]
Ok. I found one: https://github.com/ApiGen/ApiGen See what you think of that. I will spend an hour or so when I get time (tomorrow?) retro fitting our methods to use the supported annotations (https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md)

**Comment by devinpearson** [comment]
Cool yeah im all for that.

I would also like to make this a composer package as well.

Ill try do the renaming today

**Comment by harrisonhjones** [comment]
I'm all for a composer package :+1:
